gatsby_fp/gatsby-node.js

```diff
@@ -1,4 +1,42 @@
+require("dotenv").config()
 const path = require("path")
+const { createRemoteFileNode } = require("gatsby-source-filesystem")
+
+exports.createSchemaCustomization = ({ actions }) => {
+  const { createTypes } = actions
+  createTypes(`
+    type MarkdownRemark implements Node {
+      facultyImg: File @link(from: "facultyImg___NODE")
+    }
+  `)
+}
+
+exports.onCreateNode = async ({
+  node,
+  actions: { createNode },
+  store,
+  cache, 
+  createNodeId
+}) => {
+
+  if (node.internal.type === "profiles" && node.photo_url) {
+    let fileNode = await createRemoteFileNode({
+      url: node.photo_url,
+      parentNodeId: node.id,
+      createNode, // helper function in gatsby-node to generate the node
+      createNodeId,
+      cache, // Gatsby's cache
+      store, // Gatsby's Redux store
+      auth: {htaccess_user: process.env.PEOPLEAPI_USER, htaccess_pass: process.env.PEOPLEAPI_PASS}
```

Comment:
The `photo_url` itself has the credentials, do you believe we need to include it here as well?

Reply:
I'll try it without this (it should be fine without it), this is mainly from when we couldn't figure out why the build was failing (before the API was optimized for the images) so I was trying out different things on the front end lol

Reply:
Got rid of this